src/modal/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View, TouchableOpacity, Text } from 'react-native';
```

Comment:
remove the created button to use the global button component located inside the component folder when you pull again